cmake/ffig.cmake

```diff
@@ -55,11 +55,25 @@ function(ffig_add_library)
     set(ffig_invocation "${ffig_invocation};_cpp.h.tmpl")
     set(ffig_outputs "${ffig_outputs};${ffig_output_dir}/${module}_cpp.h")
   endif()
+  if(ffig_add_library_GO)
+    set(ffig_invocation "${ffig_invocation};go.tmpl")
+    set(ffig_outputs "${ffig_outputs};${ffig_output_dir}/src/${module}/${module}.go")
+  endif()
+
+  if(ffig_add_library_GO)
```

Comment:
This repetition is pretty ugly, but it's the only thing I could get to work. I welcome any and all suggestions on how to improve this!